### PR TITLE
Part of TARDIS PR

### PR DESCRIPTION
**Pull request by DalekCraft2**
See the aforementioned for details.

I managed to change the names of files in this, and I plan to do it for the two main plugins, as well.

### CODE REVIEW

File: src/main/java/me/eccentric_nz/tardisvortexmanipulator/command/TvmCommand.java [hunk 1]
```diff
@@ -66,7 +67,7 @@ public boolean onCommand(@NotNull CommandSender sender, Command command, @NotNul
                 return true;
             }
             ItemStack itemStack = player.getInventory().getItemInMainHand();
-            if (itemStack.hasItemMeta() && itemStack.getItemMeta().hasDisplayName() && itemStack.getItemMeta().getDisplayName().equals("Vortex Manipulator")) {
+            if (itemStack.hasItemMeta() && itemStack.getItemMeta().getPersistentDataContainer().get(TardisVortexManipulatorPlugin.plugin.getItemKey(), PersistentDataType.STRING).equals("vortex_manipulator")) {
```
Comment (eccentricdevotion) [inline src/main/java/me/eccentric_nz/tardisvortexmanipulator/command/TvmCommand.java]: You just broke every Vortex Manipulator created before adding this...

File: src/main/java/me/eccentric_nz/tardisvortexmanipulator/TvmRecipe.java [hunk 2]
```diff
@@ -53,11 +54,12 @@ public ShapedRecipe makeRecipe() {
             itemStack = new ItemStack(material, amount);
         }
         ItemMeta itemMeta = itemStack.getItemMeta();
-        itemMeta.setDisplayName("Vortex Manipulator");
+        itemMeta.setDisplayName(ChatColor.RESET + "Vortex Manipulator");
```
Comment (eccentricdevotion) [inline src/main/java/me/eccentric_nz/tardisvortexmanipulator/TvmRecipe.java:57]: Why? This is implicit anyway.

Comment (DalekCraft2) [reply src/main/java/me/eccentric_nz/tardisvortexmanipulator/TvmRecipe.java:57]: The italics were ugly. I think the same with every TARDIS item, but I am not going to change those. However, I _can_ do it for the GUI icons, with no side effects.

File: src/main/java/me/eccentric_nz/tardisvortexmanipulator/command/TvmCommand.java (hunk 1 above)
Comment (DalekCraft2) [reply src/main/java/me/eccentric_nz/tardisvortexmanipulator/command/TvmCommand.java]: yeah, so maybe i can figure out a way to add PBVs to all legacy ones

**Comment by eccentricdevotion** [comment]
Closing this due to reasons as stated in previous comments, here and on other PRs in this series.
Please feel free to submit parts of this as separate PRs, but please test before submitting so that only the commits necessary are included in a clean fork.

**Pull request closed without merging**